rs/registry/canister/src/mutations/do_update_subnet.rs

```diff
@@ -1076,20 +1069,85 @@ mod tests {
             &btreemap!(*first_node_id => first_dkg_pk.clone()),
         ));
 
+        (registry, subnet_id)
+    }
+
+    #[test]
+    #[should_panic(
+        expected = "Proposal attempts to change sev_enabled for Subnet 'ge6io-epiam-aaaaa-aaaap-yai' \
+             to true, but sev_enabled can only be set during subnet creation."
+    )]
+    fn test_sev_enabled_cannot_be_changed_to_true() {
+        let (mut registry, subnet_id) = make_registry_for_update_subnet_tests();
+
         let mut payload = make_empty_update_payload(subnet_id);
-        payload.features = Some(
-            SubnetFeatures {
-                canister_sandboxing: false,
-                http_requests: false,
-                sev_enabled: true,
-            }
-            .into(),
-        );
+        payload.features = Some(SubnetFeaturesPb {
+            canister_sandboxing: false,
+            http_requests: false,
+            sev_enabled: Some(true),
+        });
 
-        // Should panic because we are changing SubnetFeatures
+        // Should panic because we are changing SEV-related subnet features.
         registry.do_update_subnet(payload);
     }
 
+    #[test]
+    #[should_panic(
+        expected = "Proposal attempts to change sev_enabled for Subnet 'ge6io-epiam-aaaaa-aaaap-yai' \
+             to false, but sev_enabled can only be set during subnet creation."
+    )]
+    fn test_sev_enabled_cannot_be_changed_to_false() {
+        let (mut registry, subnet_id) = make_registry_for_update_subnet_tests();
+
+        let mut payload = make_empty_update_payload(subnet_id);
+        payload.features = Some(SubnetFeaturesPb {
+            canister_sandboxing: false,
+            http_requests: false,
+            sev_enabled: Some(false),
```

Comment:
Maybe, highlight that this is the only difference between this test and the previous one.